The timeout of 100 ms triggers when enabling the regulator of
the goodix touchscreen.

--- a/drivers/bus/sunxi-rsb.c
+++ b/drivers/bus/sunxi-rsb.c
@@ -284,7 +284,7 @@
 	       rsb->regs + RSB_CTRL);
 
 	if (!wait_for_completion_io_timeout(&rsb->complete,
-					    msecs_to_jiffies(100))) {
+					    msecs_to_jiffies(400))) {
 		dev_dbg(rsb->dev, "RSB timeout\n");
 
 		/* abort the transfer */
